app/forms/declaration/DeclarationAdditionalActors.scala

```diff
@@ -43,16 +48,18 @@ object DeclarationAdditionalActors extends DeclarationPage {
 
   val formId = "DeclarationAdditionalActors"
 
-  val mapping = Forms.mapping(
+  val mapping: Mapping[DeclarationAdditionalActors] = Forms.mapping(
     "eoriCS" -> eoriMappingFor(PartyType.Consolidator),
     "eoriMF" -> eoriMappingFor(PartyType.Manufacturer),
     "eoriFW" -> eoriMappingFor(PartyType.FreightForwarder),
     "eoriWH" -> eoriMappingFor(PartyType.WarehouseKeeper),
-    "partyType" -> mandatory(text().verifying("supplementary.partyType.error", isContainedIn(allowedPartyTypes + "no")))
+    "partyType" -> requiredRadio("declaration.partyType.error")
+      .verifying("declaration.partyType.error", isContainedIn(allowedPartyTypes + "no"))
```

Comment:
What is the purpose of `"no"` here ?

Reply:
"No" is a valid choice.

Reply:
So maybe it could be added to `allowedPartyTypes`, or at least to `PartyType` ?

Reply:
There are two situations: 1) when I need to check if I can add/validate an eori, and 2) when I need to see if all radios choices are valid. I would need two collections, but I opted for one and the second being a simple result from the first.